activities/Moon.activity/css/activity.css

```diff
@@ -52,3 +52,131 @@ canvas {
   visibility: hidden;
   z-index: 5;
 }
+
+.tutorial-title {
+	background-color: #808080 !important;
```

Comment:
Could you use the same indentation than the rest of the file (white space here)?